chart/templates/flower/flower-deployment.yaml

```diff
@@ -81,19 +81,36 @@ spec:
               containerPort: {{ .Values.ports.flowerUI }}
           livenessProbe:
             failureThreshold: 10
-            httpGet:
-              path: /
-              port: {{ .Values.ports.flowerUI }}
+            exec:
+              command:
+                - curl
+                {{- if or .Values.flower.secretName .Values.flower.basicAuth }}
+                - "-H"
+                - "Authorization: Basic $AIRFLOW__CELERY__FLOWER_BASIC_AUTH"
```

Comment:
I am afraid it will not work. The Basic Auth header is in `Authorization: Basic Base64(username:password)` format.  You can use ``--user`` parameter instead. See: https://airflow.readthedocs.io/en/latest/security/api.html#basic-authentication

Reply:
I also wonder how we can handle multiple accounts in an environmental variable.  

> provide user:password pairs separated by a comma.

https://airflow.readthedocs.io/en/latest/security/flower.html

The easiest solution is to change the values.yml file so that it only supports one account.
```
flower:
    username: dummy-username
    password: dummy-password
```